docs/modules/ROOT/pages/main/roadmap.adoc

```diff
@@ -1,98 +1,102 @@
-
-== Language Evolution (estimated 24.1)
-When using MPS, model migrations are made easy using the migration aspect. However, once stepping outside of MPS, migrations become more difficult. Model migration needs to be carried out across platforms, teams and (web) applications when a new language version is deployed.
-
-We want to support a generated APIs that can work with multiple versions at the same time with which we also can support meta-model versions checks for compatibility. We have some ideas we need to further refine and explore in the following releases.
-
-== Graphical Editors in the Web (estimated 24.2)
-The experience from MPS shows that the requirements for a graphical editor are too diverse to support all of them with a single framework. There are many different frameworks available in the web world where you can choose the one that fits your use case.
+In the following we shortly list these key priorities.
+Below you can find more detailed explanations for each of these.
+
+// ------------- 24.1
+* *ModelQL*: Dynamic query language for on-demand loading overcoming issues associated with upfront data retrieval and optimizing performance
+* *MPS 2023 / 2024 Support*: Provide support in modelix for newer MPS versions
+* *LIonWeb* (estimated 24.1): Integration with other web components via a new LionWeb bulk API adapter
+// ------------- 24.2
+* *Model-server services*: Streamlined interfaces and structure for services working on model data, providing for example model analysis, checking, and transformation
+* *Meta-model in model-server*: A way to store the meta-model of corresponding model content in the model-server (foundation for Language Evolution).
+// ------------- 25.1
+* *GraphQL interface*: An endpoint for smaller web applications, leveraging the available tooling ecosystem
+* *Projectional text editors in browser*: Projectional text editors for browsers, using Kotlin multiplatform to edit models directly or through server-side rendering
+// ------------- 25.2
+* *Graphical Editors*: We plan to provide the data synchronization and the generation of a TypeScript model API to enable graphical editors on the wev.
+
+
+// ------------- 24.1
+[#modelql,reftext="ModelQL"]
+=== ModelQL (estimated 24.1)
+
+With the modelix 23.2 release we https://modelix.org/blog/2023/11/02/modelix-platform-release-23.2/#modelql[introduced ModelQL]:
+A dynamic way of loading parts of the model on demand while reducing the number of requests to a minimum.
+To learn more, head over to the https://docs.modelix.org/modelix/main/core/explanation/modelql.html[ModelQL documentation] and the corresponding https://docs.modelix.org/modelix/main/core/howto/modelql.html[How-Tos].
+While the early version allowed us to provide a quick win for large models, we were not able to finish this feature fully.
+Especially https://docs.modelix.org/modelix/main/core/howto/modelql-writing.html[writing support] currently needs some further work.
```

Comment:
I think the writing support is pretty much done for now.
The functionality of modelql is there, but we should work on stuff like the debugging experience.